shell/platform/fuchsia/flutter/vulkan_surface.h

```diff
@@ -134,9 +134,9 @@ class VulkanSurface final
   // |EntityNode| associated with the retained surface instead of using the
   // retained surface directly. Hence Flutter can't modify the surface during
   // retained rendering.
-  const scenic::EntityNode& GetRetainedNode() {
+  scenic::EntityNode* GetRetainedNode() {
```

Comment:
The comment of this method says that we want to prevent Flutter from modifying the surface. Is changing `const scenic::EntityNode&` to `scenic::EntityNode*` breaking that promise?

Reply:
Yes. Updating comment.